hazelcast-client/src/test/java/com/hazelcast/client/map/impl/nearcache/MapNearCacheInvalidationFromClientTest.java

```diff
@@ -192,7 +192,7 @@ public void run() {
         assertTrueEventually(new AssertTask() {
```

Comment:
I was just fixing test failures due to the replacement. The whole Near Cache tests are in a quite bad state and will need more time to unify and polish. I spoke to Jaromir about this and we'll have a look that we do this after the upcoming Near Cache performance optimizations.